packages/cerebral-debugger/package.json

```diff
@@ -1,38 +1,60 @@
 {
-  "name": "cerebral-debugger",
-  "version": "2.0.0",
+  "name": "electron-quick-start",
+  "version": "1.0.0",
   "private": true,
-  "description": "Cerebral Debugger",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/cerebral/cerebral.git"
+  "description": "A minimal Electron application",
+  "main": "electron/main.js",
```

Comment:
should we have this main here or one from electron/package.json should be enough?